src/libraries/System.Private.CoreLib/src/System/Int128.cs

```diff
@@ -2192,5 +2160,27 @@ static bool INumberBase<Int128>.TryConvertToTruncating<TOther>(Int128 value, [Ma
 
         /// <inheritdoc cref="IUnaryPlusOperators{TSelf, TResult}.op_UnaryPlus(TSelf)" />
         public static Int128 operator +(Int128 value) => value;
+
+        //
+        // IBinaryIntegerParseAndFormatInfo
+        //
+
+        static bool IBinaryIntegerParseAndFormatInfo<Int128>.IsSigned => true;
+
+        static bool IBinaryIntegerParseAndFormatInfo<Int128>.IsUnsigned => false;
```

Comment:
Why do we need both IsSigned and IsUnsigned?

Reply:
It's consistent with how we model the APIs on the generic math interfaces in general.

It not only helps with readability, as it covers the affirmative case; but it also covers scenarios where `!x` does not necessarily mean `y`. For example, we have `INumberBase<TSelf>.IsPositive` and `INumberBase<TSelf>.IsNegative` because a type may be one, the other, or neither (such as is the case for `Complex` numbers).

Reply:
FWIW I think `IsSigned` and `!IsSigned` are readable.  And presumably there's no case where both IsSigned/IsUnsigned would be true nor a case we care about where both could be false.  I get the consistency with the interfaces, but we have a lot more leeway for internal stuff.

Reply:
Fixed.